### PR TITLE
Add Quest

### PR DESCRIPTION
**Pull request by AaronV77**
Adding Quest package to the conda-forge channel.

### CODE REVIEW

**Comment by AaronV77** [comment]
@conda-forge/staged-recipes can you please help me understand the above error? Thanks.

File: quest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{% set version = "2.4.1" %}
+
+package:
+  name: quest
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/erdc/quest.git
+  git_rev: {{ version }}
```
Comment (CJ-Wright) [inline quest/meta.yaml]: please build from a tarball https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

File: quest/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+about:
+  home: https://github.com/erdc/quest
+  license: BSD
+  summary: Quest is a python library that provides an API the ability to search, 
+  publish and download data (both geographical and non-geographical) across multiple 
+  data sources including both local repositories and web based services. The library 
+  also allows provides tools in order to manipulate and manage the data that the user 
+  is working with.
+  license_family: BSD
```
Comment (CJ-Wright) [inline quest/meta.yaml]: Needs a license file.

File: quest/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+package:
+  name: quest
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/erdc/quest.git
+  git_rev: {{ version }}
+
+build:
+  number: 0
```
Comment (CJ-Wright) [inline quest/meta.yaml]: Could this be noarch?

File: quest/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
```
Comment (CJ-Wright) [inline quest/meta.yaml]: Should this be `build`?

File: quest/meta.yaml (hunk 1 above)
Comment (AaronV77) [reply quest/meta.yaml]: Why does it have to be built from a tarball? Your documents say that you can use git for a build here: http://192.168.1.1/ui/1.0.99.185832/dynamic/login.html.

**Comment by jakirkham** [comment]
Also please move this to the `recipes` directory. Thanks in advance. :)

File: quest/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    - quest
+
+about:
+  home: https://github.com/erdc/quest
+  license: BSD
+  summary: Quest is a python library that provides an API the ability to search, 
+  publish and download data (both geographical and non-geographical) across multiple 
+  data sources including both local repositories and web based services. The library 
+  also allows provides tools in order to manipulate and manage the data that the user 
+  is working with.
```
Comment (jakirkham) [inline quest/meta.yaml]: Please pick a [YAML text wrapping format]( https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines/21699210#21699210 ). There are many to choose from (unfortunately this is not one). That said, they are all pretty easy to do and should be minor tweaks on what you have currently.

Comment (CJ-Wright) [reply quest/meta.yaml]: There is an example of this in the example, too! https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L65

**Pull request closed without merging**